modules/components/src/AdvancedFacetView/index.js

```diff
@@ -124,7 +124,7 @@ export default class AdvancedFacetView extends React.Component {
                     op: 'in',
                     content: {
                       field: path,
-                      value: typeof value === 'string' ? [value] : value,
+                      value: !Array.isArray(value) ? [value] : value,
```

Comment:
for some reason I'm tempted to suggest `[].concat(value)` which would work.. but that now I think that's a dumb js thing

Reply:
hum... yeah that's interesting but I think I would be confused seeing that

Reply:
one difference is you can do `[].concat(value || [])` which means if `value === null` the result is `[]` instead of `[null]`
Positive? Negative? you decide!

Reply:
or filter(Boolean)

Reply:
/shrug